demo/src/index.js

```diff
@@ -203,3 +203,6 @@ function Demo() {
 }
 
 render(<Demo/>, document.querySelector('#root'));
+
+//For usage with Storybook (temporary)
```

Comment:
What does this comment mean? Is it temporary because we will make a new "Demo" component for storybook instead of using the existing one?

Reply:
I had added lines 207 & 208 in this [commit](https://github.com/accordproject/cicero-ui/pull/382/commits/f08ec5c0dda94393d63bd50bb319797ca0dedfdf). It was added to export the Demo component to be used with storybook for testing. 

I have now reverted the changes (i.e. I have now removed line 207 & 208) now as Demo is no longer needed in Storybook.